src/main/java/frc/robot/subsystems/Turret.java

```diff
@@ -41,11 +47,34 @@ public double getVel() {
 
     public void stop() {
         turret.stopMotor();
+    }   
+
+    public void setPos(int position){
+        turret.setSelectedSensorPosition(position);
+    }
+
+    public boolean found(){
+        return RobotContainer.limelight.canSeeTarget() && RobotContainer.limelight.getTargetArea() > 0.8;
+    }
+
+    public boolean onTarget(){
+        return found() && RobotContainer.limelight.getHorizontalAngle() < 1.5;
+    }
+
+    //found online
```

Comment:
So I don't think we need to do this math here. Assuming both of these are in degrees (you can set them up so they should be) we only need to check the rotational limits of the turret.